### DISCUSSION TITLE
How can I debug the post response sent to the AccessTokenURL?

### DISCUSSION THREAD

**Discussion by denik1981** [nextauthjs/next-auth#3281] [Help]
I need to check if the body of that response is well constructed cause I'm having errors there with my custom Provider and I'm not sure where to look. The "debug" flag shows me the URL but not the body of the response.

**Comment by balazsorban44** [comment]
have you checked the network tab? 🤔

**Reply by denik1981** [reply]
But that one (AccessToken) is a server side request. Isn't it?

**Reply by balazsorban44** [reply]
Ah, you are right! I'll check out if we can improve logging there as well.

**Reply by balazsorban44** [reply]
As a workaround, you can try to log `params` in node_modules here: https://github.com/nextauthjs/next-auth/blob/35ee608d594c6a63d2236df9015e25fb1198a925/src/core/lib/oauth/callback.ts#L88-L99
